backend-controller/entryController.js

```diff
@@ -12,36 +10,93 @@ class DiaryController {
     res.send('Andela BootCamp Challenge 2.....Welcome to My Diary Api Endpoints Written by Obuladike Chisom.....');
   }
 
-  static createEntry(req, res) {
+  static createEntry(req, res, next) {
     const { error } = validateEntry(req.body);
-    if (error) res.status(400).json({ error: error.details[0].message });
-    const id = allEntries.length + 1;
-    const entryAdded = new CreateEntry(id, req.body.title, req.body.category, req.body.sub_category,
-      req.body.content);
-    allEntries.push(entryAdded);
-    res.status(201).json({ success: 'success', entry: entryAdded });
-  }
-
-  static editEntry(req, res) {
-    const entryToModify = allEntries.find(e => e.id === parseInt(req.params.id, 10));
-    if (!entryToModify || entryToModify === undefined) {
-      res.status(404).json({
-        error: 'The entry you wish to modify must have been removed or have not been created',
+    if (error) {
+      res.status(400).json({ error: error.details[0].message });
+      return;
+    }
+    const check = auth.verifyUserToken(req);
```

Comment:
@obulaworld  Based on our review yesterday, I thought we agreed that you would take all validation to the middleware, why are you still doing this here?

Reply:
okay....u know we agreed i will merge the current branch then create a refactoring branch